src/cryoER/run_cryoER_mcmc.py

```diff
@@ -162,7 +162,10 @@ def run_cryoER_mcmc(
     json_filename = "%s/Dmat.json" % (outdir)
     stan_output_file = "%s/Stan_output" % (outdir)
 
-    norm = 0.5 / (lmbd**2)
+    if lmbd == -1:
```

Comment:
I'd suggest adding documentation for this, but it doesn't look like there's any API doc to begin with.

Reply:
Ok, I can document the whole function and also the others as I get to using them